src/main/java/picard/util/IntervalListTools.java

```diff
@@ -396,28 +424,38 @@ protected int doWork() {
                 resultIntervals = scattered;
             }
         } else {
-            resultIntervals = Arrays.asList(output);
+            resultIntervals = Collections.singletonList(output);
         }
 
-        long totalUniqueBaseCount = 0;
+        long totalBaseCount = 0;
         long intervalCount = 0;
         for (final IntervalList finalInterval : resultIntervals) {
-            totalUniqueBaseCount += finalInterval.getUniqueBaseCount();
-            intervalCount += finalInterval.size();    
+            totalBaseCount += finalInterval.getBaseCount();
+            intervalCount += finalInterval.size();
         }
 
-        LOG.info("Produced " + intervalCount + " intervals totalling " + totalUniqueBaseCount + " unique bases.");
-        
+        LOG.info("Produced " + intervalCount + " intervals totalling " + totalBaseCount + " bases.");
+        switch (OUTPUT_VALUE) {
```

Comment:
You can avoid the switch and `throw` by putting this in the enum itself. It's a little more code but it's nice to avoid a switch, IMO.
```
    enum Output {
        NONE() {
            void output(long totalBaseCount, long intervalCount) {
            }
        },
        BASES {
            void output(long totalBaseCount, long intervalCount) {
                System.out.println(totalBaseCount);
            }
        },
        INTERVALS {
            void output(long totalBaseCount, long intervalCount) {
                System.out.println(intervalCount);
            }
        };
        abstract void output(long totalBaseCount, long intervalCount);
    }
...
        OUTPUT_VALUE.output(totalBaseCount, intervalCount);
```